tests/conda_env_file_generator_test.py

```diff
@@ -135,7 +135,7 @@ def test_variant_specific_env_files():
                    "--cuda_versions", "11.2",
                    "--repository_folder", os.path.join(tmp_test.name, "repos"),
                    "--output_folder", os.path.join(tmp_test.name, "output"),
-                   "https://raw.githubusercontent.com/open-ce/open-ce/open-ce-v1.5.0/envs/opence-env.yaml"]
+                   "https://raw.githubusercontent.com/open-ce/open-ce/open-ce-v1.5.0/envs/tensorflow-env.yaml"]
```

Comment:
Is this change intentional?

Reply:
Yes, this is intentional change. This test case was failing as open-ce-v1.5.0 has one remote feedstock in dali-env.yaml without `git_tag`. So, that can't be fixed now. Hence, I changed it to different env file. The test case is anyway to verify if conda env files are generated for all the combinations of variants.

Reply:
Okay. Thanks for the clarification.